### PR TITLE
JIT support for more date/time functions

### PR DESCRIPTION
**Pull request by mihaibudiu**
This is a follow up from the recent fixes to the JIT.

### CODE REVIEW

File: sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java [hunk 1]
```diff
@@ -39,10 +38,4 @@ public void timestampAddTableTest() {
                 new DBSPTimestampLiteral(600100)
         );
     }
-
-    @Test @Ignore("IString parsing not supported https://github.com/feldera/feldera/issues/338")
-    public void castTimestampToStringToTimestamp() {
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java:44]: Why are these tests being removed?

**Review by lalithsuresh — APPROVED** [review]

File: sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java (hunk 1 above)
Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java:44]: They were ignored. Not any longer

Comment (lalithsuresh) [reply sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java:44]: But it looks like the tests are removed, not just being un-ignored?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/jit/JitTimeTests.java:44]: The inherited test was ignored. It is still inherited.

**Pull request merged**